org.gnome.Evince.json

```diff
@@ -12,7 +12,10 @@
         "--filesystem=host",
         "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
         "--metadata=X-DConf=migrate-path=/org/gnome/evince/",
-        "--env=GDK_PIXBUF_MODULE_FILE=/app/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache",
+        "--filesystem=xdg-run/dconf",
```

Comment:
The DConf migration should be a separate commit.

Reply:
Would it even work if I bump the version without this line?

Reply:
Oh that's a redundent one